POLMED/src/styles/components/_building_photo.scss

```diff
@@ -15,3 +15,9 @@
 .building-photo-section-building {
   height: 50vh;
 }
+
+@media (max-width: 768px) {
```

Comment:
dodaj minimalnie większy gap pomiędzy warningiem, a górną sekcją, a tak to jest spoczko :)